staff/views/batch.py

```diff
@@ -84,7 +84,9 @@ def post(self, request):
                         capacity=sections * section_capacity,
                         start_date=batch_form.cleaned_data.get('start_date'),
                         end_date=batch_form.cleaned_data.get('end_date'),
-                        sections=sections
+                        sections=sections,
+                        price=settings.CODING_BASICS_REGISTRATION_FEE_SGD,
```

Comment:
Will we need to refactor this when we incorporate Bootcamp to Learn? Nothing to change now, just wondering.